src/node_buffer.cc

```diff
@@ -9,8 +9,21 @@
 #include "v8.h"
 
 #include <string.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <errno.h>
 #include <limits.h>
 
+#ifdef _WIN32
+  #define strtoll _strtoi64
+  #define strtoull _strtoui64
+  #define PRId64 "lld"
+  #define PRIu64 "llu"
+#else
+  #define __STDC_FORMAT_MACROS
+  #include <inttypes.h>
+#endif
```

Comment:
@bnoordhuis Are there any other platforms where this may be an issue? e.g. Darwin or BSD.